odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -119,6 +127,53 @@ message AddJobSpecificationsResponse {
   string log = 2;
 }
 
+message JobInspectRequest {
+  string project_name = 1;
+  string namespace_name = 2;
+  string job_name = 3;
+  JobSpecification spec = 4;
+}
+
+message BasicInfoSection {
+  JobSpecification job =1;
+  repeated string source =2;
+  string destination =3;
+  repeated Log notice =4;
+}
+
+message OptimusDependency {
+  string name = 1;
+  string host = 2;
+  string project_name = 3;
+  string namespace_name = 4;
+  string task_name = 5;
```

Comment:
what is this task_name? how is it different from name? Want to understand its relevance in dependency?

Reply:
cc : @arinda-arif

Reply:
Lets rename to JobDependency

Reply:
task name is name of the scheduler task , vs name is job name

Reply:
there is already a `JobDependency` in proton, but that does not capture as much into